roles/deploy/handlers/main.yml

```diff
@@ -57,3 +57,10 @@
   command: bash -lc "bundle exec whenever --set 'environment={{ rails_env }}' --update-crontab"
   args:
     chdir: "{{ current_path }}"
+
+- name: notify slack
+  slack:
+    token: "T02G54U79/BF25P9F7A/DJdtYaLLUpRJPiu72d8NqgGg"
+    msg: '`{{ git_version }}` deployed to {{ inventory_hostname }}'
+    channel: "#devops-notifications"
+    username: "ansible executed by {{ lookup('env','USER') }}"
```

Comment:
I'm not sure about this. It's a nice idea, but does anyone use a clear rendering of their own name as their username on their local machine? If we start using Semaphore it'll be `ofn-deployment`, otherwise random usernames people use on their laptops...